src/components/navigation/Breadcrumbs/BreadcrumbAction.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { colors } from '../../theme';
+
+class BreadcrumbAction extends React.Component {
+  state = { hovering: false };
+
+  render() {
+    const hover = () => {
+      this.setState({ hovering: true });
+    };
+    const unhover = () => {
+      this.setState({ hovering: false });
+    };
+    const lineStyle = this.state.hovering ? 'underline' : 'none';
+    return (
+            <span
+              onClick={this.props.Action}
+              className={this.props.ActionClasses}
+              onMouseOver={hover}
+              onMouseOut={unhover}
+              style={{ textDecorationLine: lineStyle, color: colors.accent }}

```

Comment:
Take a look at [styled-components](https://www.styled-components.com/). We use that package in several places in this project.